scripts/main/build.js

```diff
@@ -45,15 +45,17 @@ build.getAlbumThumb = function (data) {
 	var thumb2x = "";
 
 	if (thumb === "uploads/thumb/" && isVideo) {
-		return `<span class="thumbimg"><img src='img/play-icon.png' alt='Photo thumbnail' data-overlay='false' draggable='false'></span>`;
+		return `<span class="thumbimg"><img id='album-${data.id}' src='img/play-icon.png' alt='Photo thumbnail' data-overlay='false' draggable='false'></span>`;
```

Comment:
I don't think you need to add any of these `id`s to `<img>`s. The information you need can be extracted from the ancestor (`album` or `photo` `<div>`).